UK/Area/Manchester PC/PC Bandbox.prf

```diff
@@ -3,12 +3,12 @@ Settings	SettingsfilePROFILE	\..\..\Data\Settings\Local\Manchester PC\MPC_Profil
 Settings	SettingsfileSCREEN	\..\..\Data\Settings\MPC\MPC_Screen.txt
 Settings	SettingsfileVOICE	\..\..\Data\Settings\Voice.txt
 Settings	Settingsfile	\..\..\Data\Settings\MPC\MPC_General.txt
-Settings	SettingsfileARR	\..\..\Data\Settings\Lists.txt
+Settings	SettingsfileARR	\..\..\Data\Settings\Lists_SMR.txt
 Settings	SettingsfileCONFLICT	\..\..\Data\Settings\Lists.txt
-Settings	SettingsfileDEP	\..\..\Data\Settings\Lists.txt
-Settings	SettingsfileFP	\..\..\Data\Settings\Lists.txt
-Settings	SettingsfileSEL	\..\..\Data\Settings\Lists.txt
-Settings	SettingsfileSIL	\..\..\Data\Settings\Lists.txt
+Settings	SettingsfileDEP	\..\..\Data\Settings\Lists_SMR.txt
+Settings	SettingsfileFP	\..\..\Data\Settings\Lists_SMR.txt
+Settings	SettingsfileSEL	\..\..\Data\Settings\Lists_SMR.txt
+Settings	SettingsfileSIL	\..\..\Data\Settings\Lists_SMR.txt
```

Comment:
To fix

Reply:
resolved via https://github.com/VATSIM-UK/uk-controller-pack/pull/518/commits/6384cbe3e0bc8050daa8c726ba974461cc579d70